jenkins_pipelines/environments/common/pipeline-build-validation-aws.groovy

```diff
@@ -231,9 +230,9 @@ def run(params) {
                 }
             }
 
-            stage('Sanity check') {
-                sh "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'cd /root/spacewalk/testsuite; ${env.exports} rake cucumber:build_validation_sanity_check'"
-            }
+//            stage('Sanity check') {
```

Comment:
Do we miss to uncomment this?

Reply:
Sanity check is creating some chaos for now. The first objective of this pipeline is to deploy the infra. The configuration and testing part is not yet compatible. I will remove this sanity step for now. There is probably something to investigate there. The last time I used it, it took several hours to fail.